package.json

```diff
@@ -3,39 +3,60 @@
   "description": "Properly's backend logger for JavaScript",
   "license": "UNLICENSED",
   "author": "Properly Inc",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/GoProperly/js-proper-logger.git"
-  },
   "version": "0.0.1",
-  "main": "index.js",
-  "files": [],
+  "main": "dist/js-proper-logger.cjs.js",
+  "module": "dist/js-proper-logger.esm.js",
+  "types": "dist/js-proper-logger.d.ts",
   "scripts": {
+    "build": "rollup -c",
     "commit": "git-cz",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix . || true",
+    "dev": "rollup -c -w",
+    "lint": "eslint src/* test/*",
+    "lint:fix": "eslint --fix src/* test/* || true",
```

Comment:
I generally like having any "config" JS files linted as well, hence the `eslint .` default. Not too tied to it though (but curious if there was a specific reason to be more selective).

Reply:
:thumbsup: Yep. It was either "change the invocation to run on fewer things" or "update the config to ignore things" (mostly the `dist` directory). It think because this would be surprizing to someone if the added something that wasn't in the `src` or `test` directory and it wasn't linted, so I'll switch to using the `ignorePattern` approch. :thumbsup:

Reply:
Ah, adding `dist` to `eslintignore` makes sense!